manifests/s/StaticalApS/PrismforALStandardEdition(SE)/2.2.0.6/StaticalApS.PrismforALStandardEdition(SE).yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: StaticalApS.PrismforALStandardEdition(SE)

```

```suggestion
PackageIdentifier: StaticalApS.PrismforAL
```
Updating this will require changing the name of the files and folder